src/node/routes/instance.js

```diff
@@ -128,8 +128,14 @@ function defineProxyRoutes (drydock) {
           href
         } } = req;
 
+        delete headers.host;
+        let url = proxyRoute.forwardTo;
+        if (proxyRoute && proxyRoute.forwardTo instanceof Function) {
```

Comment:
Marginal preference for `if (_.isFunction(proxyRoute)) {` for consistency with the rest of the codebase.  The extra `proxyRoute &&` shouldn't be needed in that case - and will error out anyway on the previous line if it is undefined.